tests/integration/list/stack_outputs/test_stack_outputs_command.py

```diff
@@ -1,12 +1,122 @@
+        command_result = run_command(cmdlist, cwd=self.working_dir)
+        self.assertTrue(
+            re.match(
+                """^\[
+  {
+    "OutputKey": "HelloWorldFunctionIamRole",
+    "OutputValue": "arn:aws:iam::............:role/test-stack-output-exists-0-HelloWorldFunctionRole\-............",
+    "Description": "Implicit IAM Role created for Hello World function"
+  },
+  {
+    "OutputKey": "HelloWorldApi",
+    "OutputValue": "https://...........execute.*.amazonaws.com/Prod/hello/",
+    "Description": "API Gateway endpoint URL for Prod stage for Hello World function"
+  },
+  {
+    "OutputKey": "HelloWorldFunction",
+    "OutputValue": "arn:aws:lambda:.*:............:function:test-stack-output-exists-0-0-0\-HelloWorldFunction\-............",
+    "Description": "Hello World Lambda Function ARN"
+  }
+\]
+""",
+                command_result.stdout.decode(),
+            )
+        )
+
+    def test_stack_no_outputs_exist(self):
+        template_path = self.list_test_data_path.joinpath("test_stack_no_outputs_template.yaml")
+        stack_name = self._method_to_stack_name(self.id())
+        config_file_name = stack_name + ".toml"
+        region = boto3.Session().region_name
+        deploy_command_list = self.get_deploy_command_list(
+            template_file=template_path,
+            guided=True,
+            config_file=config_file_name,
+            region=region,
+            confirm_changeset=True,
+            disable_rollback=True,
+        )
+        deploy_process_execute = run_command_with_input(
+            deploy_command_list, "{}\n{}\nY\nY\nY\nY\nY\n\n\nY\n".format(stack_name, region).encode()
+        )
+        cmdlist = self.get_stack_outputs_command_list(stack_name=stack_name, region=region)
+        command_result = run_command(cmdlist, cwd=self.working_dir)
+        expected_output = (
+            f"Error: Outputs do not exist for the input stack {stack_name}" f" on Cloudformation in the region {region}"
+        )
+        self.assertIn(
+            expected_output, command_result.stderr.decode(), "Should have raised error that outputs do not exist"
+        )
+
+    def test_stack_does_not_exist(self):
+        template_path = self.list_test_data_path.joinpath("test_stack_no_outputs_template.yaml")
+        stack_name = self._method_to_stack_name(self.id())
+        config_file_name = stack_name + ".toml"
+        region = boto3.Session().region_name
+        cmdlist = self.get_stack_outputs_command_list(stack_name=stack_name, region=region)
+        command_result = run_command(cmdlist, cwd=self.working_dir)
+        expected_output = (
+            f"Error: The input stack {stack_name} does" f" not exist on Cloudformation in the region {region}"
+        )
+        self.assertIn(
+            expected_output, command_result.stderr.decode(), "Should have raised error that outputs do not exist"
+        )
+
+    def _method_to_stack_name(self, method_name):
```

Comment:
Seems this already exists in `tests/testing_utils`. Can we just use that instead or do we need this one?

Reply:
Switched to use that one.